typescript/ws-protocol-examples/README.md

```diff
@@ -1,22 +1,37 @@
-# Foxglove Studio WebSocket protocol examples
+# Foxglove WebSocket examples
 
-This package provides example server and client scripts using the [Foxglove Studio WebSocket protocol](https://github.com/foxglove/ws-protocol).
+The [Foxglove WebSocket protocol](https://github.com/foxglove/ws-protocol) supports any data encoding, as long as it is supported by both client and server. This package provides example server and client implementation of the protocol.
 
-## Usage
+## Example servers
+
+Run the `sysmon` example server, which uses JSON-encoded data to transmit system monitoring information like memory and CPU usage:
 
 ```
-$ npx @foxglove/ws-protocol-examples --help
```

Comment:
Do you think we might want to keep the `--help` line, maybe as italic/quoted text at the top of this section, just to help people discover more options for themselves?

Reply:
Added to introduction instead.